### DISCUSSION TITLE
Command to view a rule definition

### DISCUSSION THREAD

**Discussion by jaiume** [arendst/Tasmota#19627] [Q&A] [answered]
Hi everyone,

I have searched an cannot find a command that will show the definition of a rule

I know I can see the status of a rule, but I can't find a command that will give me the definition of a rule.

I have a feeling it's right in front of me, but I just can't see it

Thanks
Jamie

**Comment by barbudor** [comment, marked as answer]
Just `Rule1` without any argument to see the content of Rule1 (same for Rule2 and Rule3)
Or `Rule0` to show them all

**Comment by jaiume** [comment]
Thanks guys, 
I was trying to do it via MQTT, and I thought I tried that a few times. 

I just tried it again, and BOOM, it works

Thanks again,
Jamie
